src/ray/protobuf/gcs_service.proto

```diff
@@ -98,6 +98,16 @@ message GetNamedActorInfoReply {
   ActorTableData actor_table_data = 2;
 }
 
+message GetAllActorNamesRequest {
+  // Namespace to filter to.
+  string ray_namespace = 2;
+}
+
+message GetAllActorNamesReply {
+  GcsStatus status = 1;
+  repeated string actor_name_list = 2;
```

Comment:
How about `repeated Actor` and

```
message Actor {
  string name;
  string namespace;
}
```

Reply:
I don't currently see a use case for getting an actor name that isn't in the current namespace so I'd prefer to just scope this to the minimal version we need for now

Reply:
Isn't it important to list actors across namespaces? I don't think adding namespace changes more than a few lines of code and makes this API much nicer.

Reply:
What's the use case for listing actors across namespaces? It's sort of the whole point of namespaces to isolate these

Reply:
>What's the use case for listing actors across namespaces? It's sort of the whole point of namespaces to isolate these

I have the same question as @edoakes.

Reply:
How do I find all named actors in a cluster for debugging, or if I lost my anonymous namespace?

I think the API could be much more generally useful with just this small tweak.